plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveConfig.java

```diff
@@ -107,6 +107,7 @@
 
     private boolean bucketExecutionEnabled = true;
     private boolean sortedWritingEnabled = true;
+    private boolean optimizeSortedFileReads;
```

Comment:
I think we can enable it by default (assuming Hive maintains file order in subsequent writes)

Reply:
See https://github.com/trinodb/trino/pull/6634#discussion_r560087922

Reply:
Let's keep it disabled for now